.eslintrc.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
     'no-restricted-syntax': 'off',
     'react/prop-types': ['off'],
     'react/display-name': ['off'],
-    quotes: ['error', 'single'],
```

Comment:
We shouldn't be showing an error for using single quotes. Hope there isn't an issue with removing this.